server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/pipeline_config_representer.rb

```diff
@@ -31,7 +31,7 @@ class PipelineConfigRepresenter < ApiV1::BaseRepresenter
       end
 
       link :doc do |opts|
-        'http://api.go.cd/#pipeline-config'
+        'https://api.go.cd/current/#pipeline_config'
```

Comment:
I remember. The one where @ketan kept changing his mind. :)

I've now corrected this.